modules/aerospike/examples_test.go

```diff
@@ -0,0 +1,134 @@
+package aerospike_test
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/aerospike/aerospike-client-go/v8"
+	"github.com/testcontainers/testcontainers-go"
+	as "github.com/testcontainers/testcontainers-go/modules/aerospike"
```

Comment:
suggestion: for consistency with other modules, we usually prepend the `tc` prefix to the alias. Let's apply it elsewhere 🙏 
```suggestion
	tcaerospike "github.com/testcontainers/testcontainers-go/modules/aerospike"
```